listOrphanedDisks.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+
+#      Copyright 2015, Schuberg Philis BV
+#
+#      Licensed to the Apache Software Foundation (ASF) under one
+#      or more contributor license agreements.  See the NOTICE file
+#      distributed with this work for additional information
+#      regarding copyright ownership.  The ASF licenses this file
+#      to you under the Apache License, Version 2.0 (the
+#      "License"); you may not use this file except in compliance
+#      with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#      Unless required by applicable law or agreed to in writing,
+#      software distributed under the License is distributed on an
+#      "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#      KIND, either express or implied.  See the License for the
+#      specific language governing permissions and limitations
+#      under the License.
+
+# Script to search primary storage pools for 'orphaned disks' and remove
+# them to free up space
+
+import sys
+import getopt
+import math
+import os.path
+
+from cloudstackops import cloudstackops
+from cloudstackops import cloudstackopsssh
+from cloudstackops.cloudstackstorage import StorageHelper
+
+from prettytable import PrettyTable
+
+
+def get_volume_filesize(file_uuid_in_cloudstack, *filelist):
+    filelist, = filelist
+    size = None
+    for filepath in filelist.keys():
+        file_uuid_on_storagepool = filepath.split('/')[-1].split('.')[:1][0]
+
+        if file_uuid_in_cloudstack == file_uuid_on_storagepool:
+            size = int(filelist[filepath])
+    return size
+
+# Function to handle our arguments
+
+
+def handle_arguments(argv):
+    global DEBUG
+    DEBUG = 0
+    global DRYRUN
+    DRYRUN = 1
+    global FORCE
+    FORCE = 0
+    global zone
+    zone = ''
+    global clusterarg
+    clusterarg = ''
+    global configProfileName
+    configProfileName = ''
+
+    # Usage message
+    help = "Usage: " + os.path.basename(__file__) + ' [options] ' + \
```

Comment:
I'm literally writiing/wrote the same thing for my work place .. u pass arguments that are meaningful to the CS api such as --zone --name --networkid  ... and i pass those into the call itself ... but i for debugging and others argvs that belong to the application it self i do this sort of thing:  
dict((k, v) for k, v in res.__dict__.items() if v is not None and k !="action" and k !="debug") // to cherry pick argvs that shouldn't land on the CS call